packages/types/lib/IDataSource.ts

```diff
@@ -14,6 +14,12 @@ export interface IDataSourceExpanded {
   context?: IActionContext;
 }
 
+export interface IDataSourceRaw {
```

Comment:
To clarify that the `Raw` applies to the context, and not the datasource.
```suggestion
export interface IDataSourceRawContext {
```

Reply:
That does indeed make a lot of sense, I have refactored it in [commit 48fabfa](https://github.com/comunica/comunica/pull/1278/commits/48fabfa3b17dd4957890f69ce476c488e1980754)